demisto_sdk/commands/format/update_generic.py

```diff
@@ -220,98 +219,64 @@ def regex_matching_key(self, field, schema_keys):
                 return reg
         return None
 
-    def set_fromVersion_of_generic_object(self, from_version=None):
-        """Sets fromVersion key in a generic object file:
+    def get_answer(self, promote):
+        click.secho(promote,
+                    fg='red')
+        return input()
+
+    def ask_user(self):
+        user_answer = self.get_answer(
+            'Either no fromversion is specified in your file,'
+            ' or it is lower than the minimal fromversion for this content type, would you like to set it to the default? [Y/n]')
+        if user_answer and user_answer.lower() in ['y', 'yes']:
+            return True
+        else:
+            click.secho('Skipping update of fromVersion', fg='yellow')
+            return False
+
+    def set_default_from_version(self, default_from_version: str, current_fromversion_value: Optional[str] = None, file_type: Optional[str] = None):
```

Comment:
just noticed a tiny issue here:
If will reach this function but have a valid `current_fromversion_value`, we will still call `ask_user()` and then set `self.data[self.from_version_key]` to `current_fromversion_value` (which is already its value). please handle